pegasus/sites.v3/code.org/public/css/curriculum-pathway.scss

```diff
@@ -98,16 +98,21 @@ figure.curriculum-pathway {
       place-content: center;
       gap: 4px;
 
+      &:after {
+        color: var(--neutral_dark20);
+        font: var(--fa-font-solid);
```

Comment:
Out of curiosity, what's the difference between this syntax and using `color: $neutral_dark20`?

Reply:
I added [native css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) [here](https://github.com/code-dot-org/code-dot-org/blob/b2787fdd33ddc9452f9fe3c0f4cb08cb279b47f9/pegasus/sites.v3/code.org/styles_min/040-page.css#L9-L31) so we can use them across the code.org site on legacy pages that don't use scss, but they also work on pages that do use scss so it simplifies color variable management across the site (and has [come in extremely handy](https://github.com/code-dot-org/code-dot-org/pull/56221) recently 😅).